src/libs/actions/User.ts

```diff
@@ -541,9 +527,8 @@ function subscribeToUserEvents() {
 
 /**
  * Sync preferredSkinTone with Onyx and Server
- * @param {String} skinTone
  */
-function updatePreferredSkinTone(skinTone) {
+function updatePreferredSkinTone(skinTone: string) {
```

Comment:
Please type optimisticData, successData and failureData as OnyxUpdate[], and follow [this instructions](https://callstack-hq.slack.com/archives/C05P72RDZD2/p1695813248659689) for the API call